frontend/src/v5/ui/controls/actionMenu/actionMenu.component.tsx

```diff
@@ -64,7 +66,7 @@ export const ActionMenu = ({
 				{TriggerButton}
 			</Container>
 			<Popover
-				open={Boolean(anchorEl)}
+				open={!disabled && Boolean(anchorEl)}
```

Comment:
Using `disabled` here still triggers `onOpen`.
You can fix this by removing `disabled` here and in `handleOpen` add `if (disabled) return;` as the first line